docs/DatasetEvaluation.md

```diff
@@ -0,0 +1,211 @@
+
+Simply import the [pre-compiled pipeline](https://raw.githubusercontent.com/Unity-Technologies/datasetinsights/master/kubeflow/compiled/evaluate_the_model.yaml) into your kubeflow cluster. The figure below shows how to do this using the web UI. You can optionally use the [KFP CLI Tool](https://www.kubeflow.org/docs/pipelines/sdk/sdk-overview/#kfp-cli-tool)
+
+![upload pipeline](images/kubeflow/upload_pipeline.png)
+
+Once your pipeline has been imported, you can run it vis the web UI as shown below. Alternatively, you can use the [KFP CLI Tool](https://www.kubeflow.org/docs/pipelines/sdk/sdk-overview/#kfp-cli-tool)
+
+![evaluate the model](images/kubeflow/evaluate_the_model.png)
+
+Whether you trained a model on synthetic, real or multiple datasets, you'll need to specify a model stored in the `checkpoint_dir` from previous pipelines.
+
+You have to specify run parameters required by this pipeline:
+
+- `docker`: Path to a Docker Registry. We suggest changing this parameter to pull our images on Docker Hub with a specific tag, such as `unitytechnologies/datasetinsights:0.2.0`
+- `source_uri`: The dataset source uri. You can use the default value which points to the required dataset for this pipeline.
+- `config`: Estimator config YAML file. You can use the default value which points to a YAML file packaged with our docker images.
+- `checkpoint_file`: Path to the Estimator checkpoint file from previous training runs that you want to load for evaluation.
+- `tb_log_dir`: Path to store tensorboard logs used to visualize the training progress.
+- `volume_size`: Size of the Kubernetes Persistent Volume Claims (PVC) that will be used to store the dataset. You can use the default value.
+
+Just like for the training pipeline, you'll want to change `tb_log_dir` to point to a location that is convenient for you and your have permission to write to. This is where you'll read the logs and see the **performance metrics** once the pipeline completes.
+
+In addition to the logs, the performance metrics are also available in a Jupyter Notebook we have prepared that includes code to visualize the predictions.
+
+### visualizing predictions and performance
+
+We recommend running our [docker image](https://hub.docker.com/r/unitytechnologies/datasetinsights) which includes Jupyter as well as our notebooks if you don't want to setup the environment on your own. You can follow the same [instructions](https://github.com/Unity-Technologies/SynthDet/blob/master/docs/RunningSynthDetCloud.md#step-6-run-dataset-statistics-using-the-datasetinsights-jupyter-notebook) you may have used to run our [statistics notebook](https://github.com/Unity-Technologies/datasetinsights/blob/master/notebooks/SynthDet_Statistics.ipynb), but select the newer [visual inspection notebook](https://github.com/Unity-Technologies/datasetinsights/blob/master/notebooks/SynthDet_Evaluation.ipynb) called `SynthDet_Evaluation.ipynb` instead.
```

Comment:
Why do we need to enhance the "newer" notebook?

Reply:
removed the word `newer` as it's redundant